styles/wikiwand/catppuccin.user.css

```diff
@@ -1,5 +1,5 @@
 /* ==UserStyle==
-@name Wikiwand Catppuccin
```

Comment:
Actually we need to change `userstyles.yml` here as well:
![Screenshot 2023-12-14 at 11 46 00 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/1bd36e4d-a75e-4e69-9fc6-7b784cb58bbe)